LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2023 Ricky Padilla
```

Comment:
lol

Reply:
🙈 your legacy is everywhere don't worry!! Hahah